recipe/deploy/release.php

```diff
@@ -44,33 +36,31 @@
         return [];
     }
 
-    // Will list only dirs in releases.
-    $ll = explode("\n", run('cd releases && ls -t -1 -d */'));
-    $ll = array_map(function ($release) {
+    // retrieve all release directories
```

Comment:
Why this change here? Let’s separate into two PR.

Reply:
> @Schrank yes, I think this makes sense. Can @amayer5125 you do it as well in this pr? More context it #2759

Didn't you ask me to look at it here? Maybe I didn't need to refactor releases_list, but just releases_log to fix the issue? I'll separate it out into another PR.

Reply:
I moved the changes you mentioned to another PR (#2845).

Reply:
Thanks, got out of context. Will review it today. 

Thanks for your contributions!